osu.Game/Screens/Select/BeatmapInfoWedgeV2.cs

```diff
@@ -202,13 +208,15 @@ public partial class WedgeInfoText : Container
             public OsuSpriteText TitleLabel { get; private set; } = null!;
             public OsuSpriteText ArtistLabel { get; private set; } = null!;
 
-            public StarRatingDisplay StarRatingDisplay = null!;
+            private StarRatingDisplay starRatingDisplay = null!;
```

Comment:
These changes should have been in their own commit. They don't match the commit message.

Reply:
my bad, it was an oversight